2022-05-20 12:00:26,414 - CarbonIntensityThread - INFO - Traceback (most recent call last):
  File "/home/student/pedram-local/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/intensity.py", line 72, in carbon_intensity
    carbon_intensity = fetcher.carbon_intensity(g_location, time_dur)
  File "/home/student/pedram-local/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/fetchers/energidataservice.py", line 19, in carbon_intensity
    ci = self._emission_current()
  File "/home/student/pedram-local/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/fetchers/energidataservice.py", line 46, in _emission_current
    raise exceptions.CarbonIntensityFetcherError(response.json())
carbontracker.exceptions.CarbonIntensityFetcherError: {'statusCode': 500, 'message': 'Internal server error', 'activityId': '0600e4c0-2498-42eb-b443-4a282a402610'}

